examples/hello_world/module/rust/src/lib.rs

```diff
@@ -50,6 +53,8 @@ impl HelloWorldNode for Node {
             status.set_message("Deliberate error (code 3)".to_string());
             return Err(status);
         }
+        // Save the latest greeting to storage.
+        oak::storage::write(STORAGE_ID, STORAGE_KEY, req.greeting.as_bytes());
```

Comment:
Prefer request/response to req/rsp/res.
https://google.github.io/styleguide/cppguide.html#General_Naming_Rules